transformers4rec/torch/tabular/base.py

```diff
@@ -186,7 +187,7 @@ def from_schema(cls, schema: Schema, tags=None, **kwargs) -> Optional["TabularMo
         -------
         Optional[TabularModule]
         """
-        schema_copy = schema.copy()
+        schema_copy = deepcopy(schema)
```

Comment:
is this `deepcopy` required for the Merlin Core Schema?

Reply:
We could add a copy method to the core schema that would avoid the need for `deepcopy`

https://github.com/NVIDIA-Merlin/core/pull/240